src/Auth.ts

```diff
@@ -456,16 +456,7 @@ export class Auth {
       await logger.logToStderr('');
     }
 
-    cli.spinner.text = response.message;
-    cli.spinner.spinner = {
-      frames: ['🌶️ ']
-    };
-
-    // don't show spinner if running tests
-    /* c8 ignore next 3 */
-    if (!cli.spinner.isSpinning && typeof global.it === 'undefined') {
-      cli.spinner.start();
-    }
+    await logger.log(response.message);
```

Comment:
As this is not command output, we should move it to `stdErr`. Except when output is configured to go to `stdout`... Just like the inquirer prompt (check out `prompt.ts`)

Reply:
Good catch @martinlingstuyl 👏

Reply:
Thanks for comment. It was like this before we moved it to spinner that's why I reverted it to this state. But I will change 👍